deepmd/fit/ener.py

```diff
@@ -276,8 +276,8 @@ def _build_lower(
     ):
         # cut-out inputs
         inputs_i = tf.slice (inputs,
-                             [ 0, start_index*      self.dim_descrpt],
-                             [-1, natoms* self.dim_descrpt] )
+                             [ 0, start_index, 0],
+                             [-1, natoms, self.dim_descrpt] )
```

Comment:
dipole and polar need the same change
https://github.com/deepmodeling/deepmd-kit/blob/10fdc930d25673ac979f78c8f43148c3fad4d47f/deepmd/fit/dipole.py#L133-L141
https://github.com/deepmodeling/deepmd-kit/blob/10fdc930d25673ac979f78c8f43148c3fad4d47f/deepmd/fit/polar.py#L309-L324